Source/WebCore/Modules/WebGPU/GPUDevice.cpp

```diff
@@ -436,9 +436,10 @@ ExceptionOr<Ref<GPUBindGroup>> GPUDevice::createBindGroup(const GPUBindGroupDesc
     bool hasExternalTexture = false;
     auto* externalTexture = bindGroupDescriptor.externalTextureMatches(m_lastCreatedExternalTextureBindGroup.first, hasExternalTexture);
     if (externalTexture && (*externalTexture).get()) {
-        m_lastCreatedExternalTextureBindGroup.second->updateExternalTextures(*(*externalTexture).get());
-        RefPtr bindGroup = m_lastCreatedExternalTextureBindGroup.second.get();
-        return bindGroup.releaseNonNull();
+        if (m_lastCreatedExternalTextureBindGroup.second->updateExternalTextures(*(*externalTexture).get())) {
```

Comment:
Is it possible to fold this into one of those new fangled `if (init; condition) { ... }` I've seen you use to avoid the duplication of `(*externalTexture).get()`?

Reply:
Normal `if` worked just fine in this case, removed the duplicated `(*externalTexture).get()`, thanks for the suggestion!